pro/src/design-system/SnackBar/SnackBar.module.scss

```
@@ -0,0 +1,261 @@
+@use "styles/mixins/_a11y.scss" as a11y;
+@use "styles/mixins/_fonts.scss" as fonts;
+@use "styles/mixins/_size.scss" as size;
+
+.container {
+  position: relative;
+  width: fit-content;
+  max-width: 560px;
+
```

Généralement on utilise une fonction de conversion de **pixels -> rem** dans les utils, pour conserver des unités relatives ;)

```suggestion
@use "styles/mixins/_a11y.scss" as a11y;
@use "styles/mixins/_fonts.scss" as fonts;
@use "styles/mixins/_size.scss" as size;
@use "styles/mixins/_rem.scss" as rem;

.container {
  position: relative;
  width: fit-content;
  max-width: rem.torem(560px);

```